docs/src/main/sphinx/release/release-439.md

```diff
@@ -0,0 +1,43 @@
+# Release 439 (14 Feb 2024)
+
+## General
+
+* Fix failure when setting session properties for a catalog with a `.` in its
+  name. ({issue}`20474`)
+* Fix potential out-of-memory query failures when using the experimental scheduler. ({issue}`20694`)
+
+## BigQuery connector
+
+* Fix failure when pushing down predicates into BigQuery views. ({issue}`20627`)
+
+## Delta Lake connector
+
+* Improve performance when reading data by adding support for
+  [caching data on local storage](/object-storage/file-system-cache). ({issue}`18719`)
+* Fix potential crash when decompressing corrupted Snappy data. ({issue}`20631`)
+
+## Hive connector
+
+* Improve performance when reading data by updating support for
```

Comment:
What does "by updating support" mean?

Reply:
I think the idea is the move from Rubix to Alluxio. Maybe we need to clarify that we removed Rubix-based caching and replaed with Alluxion powered caching.

Reply:
Maybe we need one entry for Rubix removal that is a breaking change 

and then this can be the same as the others